src/components/Layout.jsx

```diff
@@ -17,7 +21,7 @@ function Layout() {
               <li className="nav-item">
                 <Link className="nav-link navbar-brand" to="/">React Crud User</Link>
               </li>
-              <li className="nav-item">
+              {/* <li className="nav-item">
```

Comment:
you can create new li instead of commenting this. if you comment this then customer crud will not show